spec/openapi.yaml

```diff
@@ -686,13 +686,13 @@ components:
           type: boolean
           example: true
 
-    FileInfo:
+    GenericFileInfo:
       type: object
       required:
         - name
-        - path
         - ro
         - type
+        - size
```

Comment:
Is this meant to also be required for folders? I think we talked about not doing it for them and also it is written in the description of size below, but in this definition it seems it is required. 

I'm not sure how precise we need to be with the definition, but theoretically it seems incorrect to me :innocent:

Reply:
size have to be optional

Reply:
Yes, my bad, it was meant only for files, but folder inherits from it. I'll fix it :)